rocketpy/mathutils/function.py

```diff
@@ -2612,6 +2622,77 @@ def compose(self, func, extrapolate=False):
                 extrapolation=self.__extrapolation__,
             )
 
+    def _check_user_input(
```

Comment:
The function doesn't access or set any attribute of the class. So I think we should convert it to a [staticmethod](https://www.geeksforgeeks.org/class-method-vs-static-method-python/). By doing this, we would allow for using the method without initializing the Function class, which may be useful if someone wants to validate the arguments

Reply:
Updated